extensions/wikia/Email/Controller/WallMessageController.class.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\EmailController;
+
+class WallMessageController extends EmailController {
+
+	/**
+	 * @template wallMessage
```

Comment:
You should use the avatarLayout template for this email type rather than define a new one.  Its meant to be reused for all emails that share this format.